proposals/dark_mode_extension_icons.md

```diff
@@ -0,0 +1,205 @@
+# Dark Mode Extension Icons
+
+**Summary**
+
+Feature to enable developers to enhance extension icon visibility in dark mode.
+
+**Document Metadata**
+
+**Author:** &lt;solomonkinard>
+
+**Sponsoring Browser:** Chromium.
+
+**Contributors:** &lt;oliverdunk>
+
+**Created:** 2024-04-05
+
+**Related Issues:**
+* https://crbug.com/893175
+* https://github.com/w3c/webextensions/issues/229
+
+## Motivation
+
+### Objective
+
+Extension developers will be able to supply and define a set of icons to be used
+in the event that browser system settings are set to dark mode. The set of dark
```

Comment:
I'm not sure that "browser system settings" is really a meaningful term. The [prefers-color-scheme spec](https://drafts.csswg.org/mediaqueries-5/#prefers-color-scheme) says dark is used if the "user has expressed the preference for a page that has a dark theme"? To avoid over-specifying here, should we say something similar?

Reply:
Done.